src/OrchardCore.Themes/TheAdmin/wwwroot/Styles/TheAdmin-darkmode.css

```diff
@@ -4,7 +4,7 @@
 */
 
 /*!
- * Bootstrap v4.5.3 (https://getbootstrap.com/)
+ * Bootstrap v4.5.2 (https://getbootstrap.com/)

```

Comment:
Why `Bootstrap v4.5.2`?

Reply:
I'm not sure why the weird things happen when I run `npm run rebuild`, how to avoid such issues?

Reply:
Happens when the package-lock.json file is not updated.

Reply:
or he did not npm install.

Reply:
I will try to run `npm install`

Reply:
@jptissot shall I run this command frequently? coz sometimes when I run the `npm run build` a lot of files tracked by git any idea

Reply:
Well, anytime @agriffard updates the dependencies you need to run npm install.

Reply:
I see .. thanks Jean, seems I need a bot coz almost the time @agriffard keep the dependencies up to date ;)

Reply:
Maybe we should modify the build scripts (npm script) to include the npm install command before ? ;)